tests/unit/markPullRequestsAsDeployedTest.ts

```diff
@@ -58,11 +66,7 @@ function mockGetInputDefaultImplementation(key) {
     }
 }
 
-/**
- * @param {String} sha
- * @returns {Promise<{data: {message: String}}>}
- */
-async function mockGetCommitDefaultImplementation({commit_sha}) {
+function mockGetCommitDefaultImplementation({commit_sha}: {commit_sha: string}): {data: {message: string}} {
```

Comment:
Let's create types for `{commit_sha: string}` and `{data: {message: string}}` to improve readability